bench/utils/__init__.py

```diff
@@ -0,0 +1,85 @@
+import struct
+import sys
+import time
+import random
+
+_RANDOM = random.Random(0)
```

Comment:
While we're here, I'm not sure that having a single instance of the RNG that is seeded once actually makes that much sense. While *nominally* that makes results reproducible, they're only reproducible if you run the exact same suite of tests. Probably good enough, but worth noting.